be/src/main/java/yeonba/be/chatting/service/ChatService.java

```diff
@@ -39,6 +47,44 @@ public class ChatService {
     private final NotificationQuery notificationQuey;
```

Comment:
```suggestion
    private final NotificationQuery notificationQuery;
```
이전 PR에서 제대로 리뷰 되지 못한 것 같네요. 죄송합니다. 오타 수정 부탁드립니다.